django/control/api/views.py

```diff
@@ -21,25 +27,147 @@
 logger = logging.getLogger(__name__)
 
 
+class BatchEndpoint(views.APIView):
+    def get(self, request):
+        batch_counter_redis = redis.Redis(
+            host=settings.REDIS_COUNTER_HOST,
+            port=settings.REDIS_COUNTER_PORT,
+            db=settings.REDIS_COUNTER_DB,
+            decode_responses=True,
+        )
```

Comment:
don't know if we should instantiate a redis client every time the route is called ? maybe we can have a connection pool shared by the API workers ?

Reply:
Workers are separated processes and will not benefit from pooling. Threads on the other hand could use pooling.
I think it's to merge as is (I don't think we'll run out of db connections) and plan to introduce pooling.